.github/workflows/deploy-preview.yml

```
@@ -0,0 +1,79 @@
+        uses: DeterminateSystems/nix-installer-action@v4
+      - name: Install Nix 2/2
+        uses: DeterminateSystems/magic-nix-cache-action@v2
+
+      - name: Prepare Sushi Lib
+        run: ./prep-sushi.sh
+
+      - name: Install Dependencies
+        run:  nix develop -c npm install
+
+      - name: Build
+        run:  nix develop -c npm run build
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)